lib/tasks/local_dev.rake

```diff
@@ -12,6 +12,8 @@ task setup_local_dev_data: %i[environment copy_feature_flags_from_production syn
   SupportUser.find_or_create_by!(dfe_sign_in_uid: 'dev-support', email_address: 'support@example.com')
 
   ProviderPermissions.update_all(manage_users: true) if FeatureFlag.active?('provider_add_provider_users')
+
+  ProviderPermissions.update_all(view_safeguarding_information: true) if FeatureFlag.active?('provider_view_safeguarding')
```

Comment:
When this runs on Heroku the flag will be inherited from production so it won't set the permission until we turn on the feature in prod — should we let it happen irrespective of flag? And above too?

Reply:
Yes good idea, it won't do any harm granting these regardless